src/commands/build.ts

```diff
@@ -368,7 +368,7 @@ export async function command(commandOptions: CommandOptions) {
   for (const proxiedFileLoc of allProxiedFiles) {
     const proxiedCode = await fs.readFile(proxiedFileLoc, {encoding: 'utf8'});
     const proxiedExt = path.extname(proxiedFileLoc);
-    const proxiedUrl = proxiedFileLoc.substr(buildDirectoryLoc.length);
+    const proxiedUrl = proxiedFileLoc.substr(buildDirectoryLoc.length).replace(/\\/g, '/'); // replace backslashes on Windows
```

Comment:
This is the solution here… but how do we feel about it?

Reply:
+1! We do this in a couple of other places, I don't hate it.

There are npm packages for converting any file path to a valid URL. In a future PR, we should look into using one of those for everthing that we do.

Reply:
Would it make sense to pull it into a util function if it's used in multiple places and would help self document with the function name?

Reply:
+1 for this. Something to mark “this is an import meant for the browser so we definitely don’t need backslashes.” We could also theoretically do things like URL escaping, but hopefully we’ll never need that.